### ISSUE TITLE
NVDA cannot read the text size input when editing it

### ISSUE THREAD

**Issue by microbit-robert** [microbit-foundation/python-editor-next#652]
In the settings dialog, the font size input is read as “blank” by NVDA when editing, although the whole line is read correctly with the value when not editing. Changing the input type to number resolves the issue, but this requires further investigation.

**Comment by microbit-robert** [comment]
The Chakra-UI component is fully accessible - https://chakra-ui.com/docs/components/form/number-input. NVDA, in my experience on chromium based browsers seems to be ignoring `aria-valuenow`, `aria-valuemin` and `aria-valuemax`. The current value is read in Firefox.

**Comment by microbit-robert** [comment]
Investigate whether this happens with the Chakra-UI example.
